src/lib/process/index.ts

```diff
@@ -36,28 +36,60 @@ export const loadProcesses = async ({
   data,
   options,
   dryRun,
+  verbose,
 }: {
   data: string
   options?: Polkadot.Options
   dryRun?: boolean
+  verbose?: boolean
 }): Promise<Process.Response> => {
   const res: Process.Response = {}
   const processes: Process.CLIParsed = JSON.parse(data)
   // TODO more elegant promise.series way.
   for (let i = 0; i < processes.length; i++) {
     const { name, version, program } = processes[i]
-    res[name] = await createProcess(name, version, program, dryRun, options)
+    res[name] = await createProcess(name, version, program, dryRun, options, verbose)
   }
 
   return res
 }
 
+export const listProcess = async (
+  res: Process.RawPayload[], 
+  processes: Process.RawPayload[], 
+  options: Process.CLIOptions) => {
+  if (options.active) {
+    processes = res.filter(({ status }) => status === 'Enabled')
+  } else if (options.disabled) {
+    processes = res.filter(({ status }) => status === 'Disabled')
+  } else {
+    processes = res
+  }
+
+  if (options.raw) {
+    return [processes, { depth: null }]
```

Comment:
Probably better off just returning the `processes` and not the `{ depth: null }`. The latter parameter is the concern of `dir` which is called elsewhere.